src/state/selections/reducer.ts

```diff
@@ -11,16 +11,18 @@ import {
     SelectionStateBranch,
 } from "./types";
 
-const typeToDescriptionMap: TypeToDescriptionMap<SelectionStateBranch, Action> = {
+type NavigationTabType = string;
```

Comment:
What do you think about moving `NavigationTabType` into `state/selections/types`? That way you could also use it within `SelectionStateBranch`